### DISCUSSION TITLE
Support Kotlin Multiplatform

### DISCUSSION THREAD

**Discussion by jakoss** [mcarleio/konvert#12] [Ideas]
I think that there shouldn't be any reason for Konvert to be locked on to JVM, but maybe there are some internals that rely on that?

**Comment by mcarleio** [comment]
Hm, I have to admit that I never did a multiplatform kotlin project and therefore am unsure what might prevent switching to kotlin multiplatform and how much I am dependant on the JVM stuff  :thinking:

**Reply by jakoss** [reply]
KMP is getting traction and KSP should theoretically enable that :)

**Comment by mcarleio** [comment]
KMP is coming to Konvert with https://github.com/mcarleio/konvert/pull/219
